apic_ml2/neutron/plugins/ml2/drivers/cisco/apic/network_constraints.py

```diff
@@ -127,14 +149,22 @@ def parse_cidr_list(cidrs):
                     net = tuple(section_name.split('/', 1))
                     if len(net) < 2:    # tenant case
                         net += tuple([None])
+                    allow_vm_names[net] = []
                     constraints[net] = {}
                     for k, v in cfg_file[section_name].iteritems():
                         k = k.lower()
                         if k in ['public', 'private', 'deny']:
                             constraints[net][k] = parse_cidr_list(v[0])
                         elif k == 'default':
                             constraints[net][k] = sanitize_scope(v[0])
-                    LOG.debug('Constraints for network %(n)s - %(c)s',
-                              {'n': net, 'c': constraints[net]})
+                        elif k == 'allow_vm_names':
+                            allow_vm_names[net] = parse_allow_vm_names(v[0])
+                    if constraints[net]:
```

Comment:
I don't understand the need for this conditional logging. Why not just log the constraints and allowed-vms always?